monitoring/ecs.tf

```diff
@@ -0,0 +1,3 @@
+resource "aws_ecs_cluster" "cluster" {
```

Comment:
Previously the cluster was loaded from an output in the catalogue_pipeline stack, but that output no longer exists.